src/scripts/main.js

```diff
@@ -1,3 +1,320 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const tableRows = tableBody.rows;
+let toggle = true;
+let valueForSort = 0;
+let special;
+
+function setSalaryValue(num) {
+  return `$${num.toLocaleString('en-US')}`;
+};
+
+function getSalaryValue(str) {
+  return +str.replace(/[,$]/g, '');
+};
+
+tableHead.addEventListener('click', e => {
+  const index = e.target.cellIndex;
+  const headTitle = e.target.innerHTML;
+
+  const sortRows = [...tableRows].sort((a, b) => {
```

Comment:
Consider wrapping the sort function in a separate named function for better readability.